sdk/eventhub/azure-eventhub-checkpointstoreblob-aio/azure/eventhub/extensions/checkpointstoreblobaio/_blobstoragecsaio.py

```diff
@@ -127,27 +129,21 @@ async def _upload_ownership(
         ownership["last_modified_time"] = uploaded_blob_properties[
             "last_modified"
         ].timestamp()
-        ownership.update(metadata)
```

Comment:
I think it's reasonable to remove this line as the ownership doesn't need to have "ownerid" ("onwer_id" is wanted), just make sure it won't break the code

Reply:
yep, removed the line and tested by installing these changes and running the tests + checkpoint store samples in `azure-eventhub`. They all pass + samples work. Since we don't document `ownerid` as a returned value in dict, it shouldn't break customers either (or if it does, they would shift to using `owner_id`).